pkg/util/utils.go

```diff
@@ -1219,3 +1219,28 @@ func ConvertTimeout(timeout int) uint {
 	}
 	return uint(timeout)
 }
+
+func ExecAddTERM(useTTY bool, existingEnv []string, execEnvs map[string]string) {
```

Comment:
Since there's only one caller and no uni tests, can we move it directly to `pkg/domain/infra/abi/containers.go`?

Reply:
There are two callers, one from api/handlers an one from domain/infra/abi